### ISSUE TITLE
Trouble booking open restaurants 

### ISSUE THREAD

**Issue by christj99** [Alkaar/resy-booking-bot#80]
I'm trying to book openly available restaurants to test the code for me. I found someone raised this issue earlier,  #50 , and picked the restaurant they chose (https://resy.com/cities/ny/the-bar-room-at-the-beekman?date=2023-01-14&seats=2). I have a payment method on my account. I noticed the config file has changed since that issue so I'm trying to troubleshoot what I could be doing wrong. thoughts?

Below is my config file

````
############
# ResyKeys #
############
# Your user profile API key. Can be found once you're logged into Resy in most "api.resy.com" network
# calls (i.e. Try they "/find" API call when visiting a restaurant). Open your web console and look for a request header
# called "authorization".
# e.g.
# resyKeys.api-key="MY_API_KEY"
resyKeys.api-key="MY KEY"
# Your user profile authentication token when logging into Resy. Can be found once you're logged into
# Resy in most "api.resy.com" network calls (i.e. Try the "/find" API call when visiting a restaurant). Open your web
# console and look for a request header called "x-resy-auth-token".
# e.g.
# resyKeys.auth-token="MY_AUTH_TOKEN"
resyKeys.auth-token="My Token"

######################
# ReservationDetails #
######################
# The date you want to make the reservation in YYYY-MM-DD format. This should be set to the day after the
# last available day with restaurant reservations as this is the day you want to snipe for a reservation once they
# become available.
# e.g.
# resDetails.date="2099-01-30"
resDetails.date="2023-01-16"
# Size of the party reservation
# e.g.
# resDetails.party-size=2
resDetails.party-size=2
# The unique identifier of the restaurant you want to make the reservation at. Can be found when viewing
# available reservations for a restaurant as a query parameter in the `/find` API call if you have the web console open.
# e.g.
# resDetails.venue-id=123
resDetails.venue-id=40703
# Priority list of reservation times and table types. Time is in military time HH:MM:SS format. This
# allows full flexibility on your reservation preferences. For example, your priority order of reservations can be...
#   * 18:00 - Dining Room
#   * 18:00 - Patio
#   * 18:15
#   If you have no preference on table type, then simply don't set it and the bot will pick a reservation for that time
#   slot regardless of the table type.
# e.g.
# resDetails.res-time-types=[
#     {reservation-time="18:00:00", table-type="Dining Room"},
#     {reservation-time="18:00:00", table-type="Patio"},
#     {reservation-time="18:15:00"}
# ]
resDetails.res-time-types=[
    {reservation-time="21:00:00"}
  ]

#############
# SnipeTime #
#############
# Hour of the day when reservations become available and when you want to snipe
# e.g.
# snipeTime.hours=9
snipeTime.hours=10
# Minute of the day when reservations become available and when you want to snipe
# e.g.
# snipeTime.minutes=0
snipeTime.minutes=37
````

Here is my output

````
sbt:resy-booking-bot> run
[info] running com.resy.ResyBookingBot
[INFO ] 2023-01-14 10:34:57.921-06:00 ResyBookingBot$:16 - Starting Resy Booking Bot
[INFO ] 2023-01-14 10:34:59.008-06:00 ResyBookingBot$:45 - Next snipe time: 2023-01-14T10:37:00.000-06:00
[INFO ] 2023-01-14 10:34:59.009-06:00 ResyBookingBot$:46 - Sleeping for 0 hours, 1 minutes, and 59 seconds
[success] Total time: 43 s, completed Jan 14, 2023 10:34:59 AM
[INFO ] 2023-01-14 10:36:58.032-06:00 ResyBookingWorkflow:18 - Taking the shot...
[INFO ] 2023-01-14 10:36:58.033-06:00 ResyBookingWorkflow:19 - (╥éΓÇ╛ Γû╡ΓÇ╛)∩╕╗πâçΓòÉΣ╕Ç (╦ÜΓû╜╦ÜΓÇÖ!)/
[INFO ] 2023-01-14 10:36:58.033-06:00 ResyBookingWorkflow:20 - Attempting to snipe reservation
[INFO ] 2023-01-14 10:37:08.059-06:00 ResyClient:176 - Missed the shot!  
[INFO ] 2023-01-14 10:37:08.060-06:00 ResyClient:177 - Γö╗ΓöüΓö╗ ∩╕╡ \(┬░Γûí┬░)/ ∩╕╡ Γö╗ΓöüΓö╗
[INFO ] 2023-01-14 10:37:08.061-06:00 ResyClient:178 - Could not find any available reservations
[INFO ] 2023-01-14 10:37:08.062-06:00 ResyBookingBot$:53 - Shutting down Resy Booking Bot
sbt:resy-booking-bot>
[info] shutting down sbt server
````

doesn't seem like I'm getting any errors so I'm hoping someone can point me in the right direction.

**Comment by Alkaar** [comment]
Assuming your API key and auth token are correct, it looks fine to me. If you flip the log4j file to debug, it will show you more details as to what it's doing. Use that to try to troubleshoot.

**Comment by jlgood23** [comment]
Hey did you end up figuring this out? Just started running mine and have same issue.

**Comment by robgreenberg3** [comment]
I am having same issue @ctcavarretta and @christj99 , lmk if you solve. It runs and even with open reservations, it is failing to book successfully.

**Comment by Alkaar** [comment]
The original issue that @christj99 raised I think is unrelated since it was from a while ago. I did some testing today and am encountering the same issue as @ctcavarretta and @robgreenberg3. I believe this started happening within the last few days as I was able to book reservations before this. This is the same issue as https://github.com/Alkaar/resy-booking-bot/issues/102 so I'm going to close this one out and direct everyone to follow that issue instead.
